src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Sofie</title>
+  <title>Alba</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="assets/sofie-logo.svg">
```

Comment:
Im guessing it's fine we leave the favicon in right? :)

Reply:
Nice catch :) I have removed it and added a TODO to readd it when we have an icon for it. There is no reason to keep it, since it is pointing to a deleted asset.